Tests/RxCocoaTests/Observable+BindTests.swift

```diff
@@ -112,13 +162,39 @@ extension ObservableBindTest {
             .next(1)
             ])
     }
+
+    func testBindToPublishRelays() {
+        var events1: [Recorded<Event<Int>>] = []
+        var events2: [Recorded<Event<Int>>] = []
+
+        let relay1 = PublishRelay<Int>()
+        let relay2 = PublishRelay<Int>()
+
+        _ = relay1.subscribe { event in
+            events1.append(Recorded(time: 0, value: event))
+        }
+
+        _ = relay2.subscribe { event in
+            events2.append(Recorded(time: 0, value: event))
+        }
+
+        _ = Observable.just(1).bind(to: relay1, relay2)
+
+        XCTAssertEqual(events1, [
+            .next(1)
+            ])
+
+        XCTAssertEqual(events2, [
+            .next(1)
+            ])
+    }
     
     func testBindToOptionalPublishRelay() {
         var events: [Recorded<Event<Int?>>] = []
         
         let relay = PublishRelay<Int?>()
         
-        _ = relay.subscribe{ event in
+        _ = relay.subscribe { event in
```

Comment:
This is not testing the variadic case, is there a need for this test and the non variadic ones?

Reply:
As written below, in my though process, yes - 
I believe our tests should always test that the existing interface and the old interface works (e.g. binding to a single one vs. binding to a variadic list).